molecule/renew/requirements.yml

```diff
@@ -1,3 +1,4 @@
 ---
 collections:
-  - community.crypto
+  - name: community.crypto
+    version: "<3.0.0"
```

Comment:
Could you add an inline comment why we fix the version?

Reply:
Done